ig-root/input/resources/StructureDefinition-disease.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/R4/fhir-single.xsd http://www.w3.org/1999/xhtml http://hl7.org/fhir/R4/fhir-xhtml.xsd">
+  <id value="condition-disease" />
+  <text>
+    <status value="generated" />
+    <h:div>
+      <h:p>The word phenotype is used with many different meanings, including ‘the observable traits of an organism’. In medicine, the word can be used with at least two different meanings. It is used to refer to some observed deviation from normal morphology, physiology, or behavior. In contrast, the disease is a diagnosis, i.e., and inference or hypothesis about the cause underlying the observed phenotypic abnormalities.</h:p>
+    </h:div>
+  </text>
+  <url value="http://github.com/phenopackets/core-ig/StructureDefinition/Disease" />
```

Comment:
This should be: http://github.com/phenopackets/core-ig/StructureDefinition/condition-disease  because it should match the id value.

Reply:
fixed